Anchorage.swift

```diff
@@ -262,214 +264,221 @@ infix operator ~: PriorityPrecedence
+#endif
 
 public struct LayoutExpression<T : LayoutAnchorType> {
-
-    fileprivate var anchor: T?
-    fileprivate var constant: CGFloat
-    fileprivate var multiplier: CGFloat
-    fileprivate var priority: UILayoutPriority
-
-    fileprivate init(anchor: T? = nil, constant: CGFloat = 0.0, multiplier: CGFloat = 1.0, priority: UILayoutPriority = UILayoutPriorityRequired) {
+    
+    var anchor: T?
+    var constant: CGFloat
+    var multiplier: CGFloat
+    var priority: UILayoutPriority
+    
+    init(anchor: T? = nil, constant: CGFloat = 0.0, multiplier: CGFloat = 1.0, priority: UILayoutPriority = UILayoutPriorityRequired) {
         self.anchor = anchor
         self.constant = constant
         self.multiplier = multiplier
         self.priority = priority
     }
-
+    
 }
 
 // MARK: - EdgeAnchorsProvider
 
 public protocol EdgeAnchorsProvider {
-
+    
     var edgeAnchors: EdgeAnchors { get }
-
+    
 }
 
 extension EdgeAnchorsProvider {
-
+    
     public var horizontalAnchors: EdgeAnchors {
-        return edgeAnchors.filter(LayoutEdge.Horizontal)
+        return edgeAnchors.filter(forEdges: LayoutEdge.Horizontal)
     }
-
+    
     public var verticalAnchors: EdgeAnchors {
-        return edgeAnchors.filter(LayoutEdge.Vertical)
+        return edgeAnchors.filter(forEdges: LayoutEdge.Vertical)
     }
-
+    
 }
 
 extension UIView: EdgeAnchorsProvider {
-
+    
     public var edgeAnchors: EdgeAnchors {
         return EdgeAnchors(top: topAnchor, leading: leadingAnchor, bottom: bottomAnchor, trailing: trailingAnchor)
     }
-
+    
 }
 
 extension UIViewController: EdgeAnchorsProvider {
-
+    
     public var edgeAnchors: EdgeAnchors {
         return EdgeAnchors(top: topLayoutGuide.bottomAnchor, leading: view.leadingAnchor, bottom: bottomLayoutGuide.topAnchor, trailing: view.trailingAnchor)
     }
-
+    
 }
 
 extension UILayoutGuide: EdgeAnchorsProvider {
-
+    
     public var edgeAnchors: EdgeAnchors {
         return EdgeAnchors(top: topAnchor, leading: leadingAnchor, bottom: bottomAnchor, trailing: trailingAnchor)
     }
-
+    
 }
 
 // MARK: - LayoutEdge
 
-private enum LayoutEdge {
-
+enum LayoutEdge {
+    
     case top, leading, bottom, trailing
     static let Horizontal = [leading, trailing]
     static let Vertical = [top, bottom]
     static let All = [top, leading, bottom, trailing]
-
-    var axis: UILayoutConstraintAxis {
+    
+    var axis: ConstraintAxis {
         switch self {
         case .top, .bottom:
             return .vertical
         case.leading, .trailing:
             return .horizontal
         }
     }
-
-    func transform(constant: CGFloat) -> CGFloat {
+    
+    func transform(constant theConstant: CGFloat) -> CGFloat {
         switch self {
         case .top, .leading:
-            return constant
+            return theConstant
         case .bottom, .trailing:
-            return -constant
+            return -theConstant
         }
     }
+    
+}
 
+enum ConstraintAxis: Int {
```

Comment:
This one is purely to deal with differing case on the first letter between enum cases in swift 2.x and 3.0. because it was all internal methods that used it I figured I'd just declare another type. An alternative might be to declare constants that point to the compatible version of UILayoutConstraintAxis (ie. `let foo = UILayoutConstraintAxis.Horizontal` vs. `let bar = UILayoutConstraintAxis.horizontal`). Definitely open to any other suggestions you might have for this compatibility issue .